sdk/component/storer.go

```diff
@@ -0,0 +1,20 @@
+package component
+
+import "github.com/smithy-security/smithy/sdk/component/internal/storer/local"
+
+type storeType string
+
+const (
+	storeTypeTest  storeType = "test"
+	storeTypeLocal storeType = "local"
+)
+
+func isAllowedStoreType(st storeType) bool {
+	return st == storeTypeLocal
+}
+
+// newStore - TODO - implement in another PR.
+func newStorer(storeType storeType) (Storer, error) {
```

Comment:
This is a potato local version for now. Will implement the proper storage in the following PR.